src/app/containers/Ad/Amp/index.jsx

```diff
@@ -80,15 +90,14 @@ const AmpAd = ({ slotType }) => {
   const label = pathOr('Advertisement', ['advertisementLabel'], ads);
 
   return (
```

Comment:
In the PR description says that the `psammead-amp-geo` has been added into this container, but can't find it. Is that still in progress or I might have missed something?

Reply:
Sorry, I have updated that. We have to reuse the amp-geo component that is being used by the rest of the components in Simorgh, as you can't have more than one on a page.

Reply:
Ahh ok 👍